samples/speech_transcribe_async.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright 2019, Google, Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+function main(
+  filepath = 'path/to/audio.raw',
+  encoding = 'LINEAR16',
+  sampleRateHertz = 16000,
+  languageCode = 'en-US'
+) {
+  // [START speech_transcribe_async]
+  /**
+   * TODO(developer): Uncomment these variables before running the sample.
```

Comment:
Is `Uncomment these variables before running the sample.` the language we always want to use?

What about the cases where samples run just fine, as-is. No modifications necessary?

We have set this as a goal, when possible, and have been quite successful. e.g. there is a GCS version of this sample where, rather than a local audio file, a path to an audio file in GCS is specified. We point to a public audio file in our common bucket for sample files.

Reply:
Love this approach, and would love to us trying to do it as much as humanly possible.

Reply:
Yes, the guidance from our working group was to minimize how many variables the user must set (with the exception of project ID mentioned in other comments, but still up for discussion).